docs/user_manual/performance-guide.md

```diff
@@ -62,7 +62,7 @@ With the introduction of columnar data input to PGM, integrating with databases
 Depending on the details of the batch, a number of performance optimizations are possible:
 
 - [Topology constructions](#topology-caching), especially, may significantly impact the computation time of a scenario.
-- The way the [batch data set](#using-independent-batches) is provided to the model can also affect the performance.
+- The way the [batch data set](#batch-data-set) is provided to the model can also affect the performance.
```

Comment:
content change

Reply:
Broken link ==>> flagged a linking error, as mentioned explicitly in the description of this PR